### PR TITLE
Code review for QuantileSketch incorporated

### PR DESCRIPTION
**Pull request by parikg**
Change to structure of ColumnSortOrder and QuantileSketch.

### CODE REVIEW

File: hieroplatform/src/test/java/org/hiero/sketch/RowsAsStrings.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import java.util.ArrayList;
 
-public class RowsAsStrings {
+class RowsAsStrings {
```
Comment (mbudiu-vmw) [inline hieroplatform/src/test/java/org/hiero/sketch/RowsAsStrings.java]: I don't know what this class is.

File: hieroplatform/src/test/java/org/hiero/sketch/TableDataSetTest.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        final ParallelDataSet<Table> par = new ParallelDataSet<Table>(elems);
+        final QuantileSketch qSketch = new QuantileSketch(cso, resolution);
+        final QuantileList r = par.blockingSketch(qSketch);
+        ListComparator comp = cso.getComparator(r.quantile);
+        for(int i = 0; i < r.getQuantileSize() -1; i++ )
+            assertTrue(comp.compare(i,i+1) <= 0);
+        //System.out.println(r);
+    }
+
+    /*
```
Comment (mbudiu-vmw) [inline hieroplatform/src/test/java/org/hiero/sketch/TableDataSetTest.java]: Maybe you can delete this

File: hieroplatform/src/test/java/org/hiero/sketch/QuantileSketchTest.java [hunk 3]
```diff
@@ -1,36 +1,54 @@
...
-        //System.out.println(mergedQ.toString());
+        final int resolution = 100;
+        final QuantileSketch qSketch = new QuantileSketch(cso, resolution);
+        final QuantileList leftQ = qSketch.getQuantile(leftTable);
+        final ListComparator leftComp = cso.getComparator(leftQ.quantile);
+        //System.out.println(leftQ);
+        for(int i = 0; i < leftQ.getQuantileSize() -1; i++ )
+            assertTrue(leftComp.compare(i,i+1) <= 0);
+        final QuantileList rightQ = qSketch.getQuantile(rightTable);
+        //System.out.println(rightQ);
```
Comment (lalithsuresh) [inline hieroplatform/src/test/java/org/hiero/sketch/QuantileSketchTest.java]: Just a nit. Please remove all commented out System.out.printlns() from all the affected files.

File: hieroplatform/src/test/java/org/hiero/sketch/TableDataSetTest.java [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        for (String colName: randTable.schema.getColumnNames()) {
+            cso.append(new ColumnOrientation(randTable.schema.getDescription(colName), true));
+        }
+        final QuantileSketch qSketch = new QuantileSketch(cso, resolution);
+        final LocalDataSet<Table> ld = new LocalDataSet<Table>(randTable);
+        final QuantileList ql = ld.blockingSketch(qSketch);
+        ListComparator comp = cso.getComparator(ql.quantile);
+        for(int i = 0; i < ql.getQuantileSize() -1; i++ )
+            assertTrue(comp.compare(i,i+1) <= 0);
+        //System.out.println(ql);
```
Comment (lalithsuresh) [inline hieroplatform/src/test/java/org/hiero/sketch/TableDataSetTest.java]: Some more commented out print statements.

**Pull request merged**